### ISSUE TITLE
Support tuple/list as `axis` argument in `BatchNormalization` layer (see implementation in `tf.keras.layers.BatchNormalization`).

### ISSUE THREAD

**Issue by gabrieldemarmiesse** [keras-team/keras#12126]
Everything is in the title.

**Comment by vcarpani** [comment]
I might look into this
